### DISCUSSION TITLE
install.sh update

### DISCUSSION THREAD

**Discussion by UcraNerd** [prasanthrangan/hyprdots#176] [Ideas]
It would be helpfull if you insert into the install script the option to chose your aspect ratio, and based one the answer make the config files more compatible with different monitors.

**Comment by prasanthrangan** [comment]
Hmm.. but I think config files (mainly rofi) has to be dynamic, rather than creating different config files for different aspect ratios.

**Reply by psilvaricardo** [reply]
I'm totally new to this setup and I have been using it for like a week now, it works and looks amazing, great job @prasanthrangan 

I saw you are actively working on this setup, is there any script I can use to update my PC with your latest changes? If so, how can I use it? How can keep my PC up to date with your repository? 

How can actually apply any updates (like the below) to my PC ?

![image](https://github.com/prasanthrangan/hyprdots/assets/125488497/814142c8-8bf2-43d5-b711-0e85cc9af48f)

**Reply by prasanthrangan** [reply]
Hi, thank you so much!

After the pull, you can just go to `Scripts` directory and rerun the install script as below -
```
./install.sh -r
```

**Reply by psilvaricardo** [reply]
Thank you for your feedback, I appreciate it @prasanthrangan 

I have been super lately, but I hope I will have time to collaborate with this amazing project in the near future.
